src/analysis/inner_product_space/linear_pmap.lean

```diff
@@ -0,0 +1,216 @@
+/-
+Copyright (c) 2022 Moritz Doll. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Moritz Doll
+-/
+
+import analysis.inner_product_space.adjoint
+import topology.algebra.module.linear_pmap
+import topology.algebra.module.basic
+
+/-!
+
+# Partially defined linear operators on Hilbert spaces
+
+We will develop the basics of the theory of unbounded operators on Hilbert spaces.
+
+## Main definitions
+
+* `linear_pmap.is_formal_adjoint`: An operator `T` is a formal adjoint of `S` if for all `x` in the
+domain of `T` and `y` in the domain of `S`, we have that `⟪T x, y⟫ = ⟪x, S y⟫`.
+* `linear_pmap.adjoint`: The adjoint of a map `E →ₗ.[𝕜] F` as a map `F →ₗ.[𝕜] E`.
```

Comment:
This isn't something to fix in this PR, but I can't easily visually distinguish `E →ₗ.[𝕜] F` from `E →ₗ⋆[𝕜] F` on my machine, which made me confused at first, so I'm just making this note to draw attention to it.